src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -744,7 +744,7 @@ public class OpcodeSetMember : Opcode
         public override void Execute(ICpu cpu)
         {
             Structure value = cpu.PopStructureEncapsulated();         // new value to set it to
-            string suffixName = cpu.PopStack().ToString().ToUpper();  // name of suffix being set
+            string suffixName = cpu.PopStack().ToString();            // name of suffix being set
```

Comment:
Alright, but then have we ensured that we changed the comparisons *everywhere*?  Because this is a universal change (i..e ALL GetMember calls everywhere no longer convert case), while each of the places you've edited ``if (foo == "STRING")`` to use OrdinalIgnoreCase are individual single cases.  Is there a danger we missed an individual single case.to correspond to this universal change?

It just makes me a little nervous because I doubt we have test cases of literally every hardcoded keyword we ever made.